src/mame/machine/genfruit.h

```diff
@@ -0,0 +1,33 @@
+// license:BSD-3-Clause
+// copyright-holders:SomeRandomGuyIdk
+#ifndef GENFRUIT_H_
+#define GENFRUIT_H_
+
+#include "sound/samples.h"
+
+#define SAMPLE_METER 0
+#define SAMPLE_PAYOUT 1
+
+const char *const genfruit_sample_names[] =
+{
+	"*genfruit",
+	"meter",
+	"payout",
+	nullptr   /* end of array */
+};
```

Comment:
Fake samples for events really isn’t the direction we want to go in.  Please don’t start doing this for fruit machines.

Reply:
Are samples very frowned upon? MFME has them, plus IMO they make the experience a bit more authentic. Also for
machines this old, in some cases payout sounds are the only feedback about winning something. Just something I'd like to keep.